### PR TITLE
Some fix and improvements

### PR DESCRIPTION
**Pull request by shurik2k5**
- dropdown list in GridView filed search filter optimization
 - select position ActionColumn in GridView (left or right)
 - fix view icon in ActionColumn
 - pager align in GridView

### CODE REVIEW

**Comment by shurik2k5** [comment]
try to fix https://github.com/schmunk42/yii2-giiant/issues/250

File: docs/20-batches.md [hunk 1]
```diff
@@ -25,8 +25,9 @@ With the config array...
             'crudViewPath' => '@project/modules/crud/views',
             'crudPathPrefix' => '/crud/',
             'crudTidyOutput' => true,
+            'crudActionButtonColumnPosition' => 'right', //left by default
             'crudProviders' => [
-                \schmunk42\giiant\generators\crud\providers\core\OptsProvider::class
+                \schmunk42\giiant\generators\crud\providers\core\OptsProvider::className()
```
Comment (schmunk42) [inline docs/20-batches.md]: No need to change this.

File: src/generators/crud/default/views/index.php [hunk 2]
```diff
@@ -208,7 +208,7 @@
                         'aria-label' => Yii::t('{$generator->messageCategory}', 'View'),
                         'data-pjax' => '0',
                     ];
-                    return Html::a('<span class="glyphicon glyphicon-file"></span>', \$url, \$options);
+                    return Html::a('<span class="glyphicon glyphicon-eye-open"></span>', \$url, \$options);
```
Comment (schmunk42) [inline src/generators/crud/default/views/index.php]: I do not favor changing the icons, CC: @eluhr WDYT?

Comment (eluhr) [reply src/generators/crud/default/views/index.php]: I think an open eye describes the target page better than a file icon. I'd change it.

Comment (schmunk42) [reply src/generators/crud/default/views/index.php]: OK :smile:

File: docs/20-batches.md (hunk 1 above)
Comment (shurik2k5) [reply docs/20-batches.md]: All generated files use old definitions className() to use with old php versions and I replace it in documentation.

File: src/generators/crud/default/views/index.php (hunk 2 above)
Comment (shurik2k5) [reply src/generators/crud/default/views/index.php]: On index.php CRUD view use file icon, but in view.php use eye icon. Change to the same style as used in the framework gii generated CRUDs

**Comment by schmunk42** [comment]
Thank you!

**Pull request merged**